src/test/ui/issues/issue-23302-3.stderr

```diff
@@ -1,26 +1,20 @@
-error[E0391]: cycle detected when const checking if rvalue is promotable to static `A`
-  --> $DIR/issue-23302-3.rs:1:1
-   |
-LL | const A: i32 = B;
-   | ^^^^^^^^^^^^^^^^^
-   |
-note: ...which requires checking which parts of `A` are promotable to static...
+error[E0391]: cycle detected when processing `A`
```

Comment:
Hmm, I think maybe `mir_const_qualif` or whatever it's called should have a message along the lines of `... const-checking A`.